docs/introduction/getting-started.md

```diff
@@ -1,6 +1,6 @@
-# Getting started
+# Getting Started
 
-The term for automatically testing, building and deploying your project is Continuous Integration, or CI for short.
+The term for automatically testing and building your project is Continuous Integration, or CI for short.
```

Comment:
I believe the explanation of CI was quite correct. I deployment is an essential part of integrating code nowadays.

Reply:
I was under the impression if you have deployment it becomes CI/CD. The initial landing page also says
> Continuous Integration is a widely-used practice where automated pipelines check the latest code changes actually work, before merging them into the main branch.

Which had no mention of deployment. I can put it back and add deployment to the main landing page if that's better.

Reply:
Indeed it has no mention of deployment, but deployment doesn't need to mean live on production. It can also mean on a test environment or as a zipfile to be inspected by some end-to-end test.

The D in CD also doesn't mean deployment. The concept is about continuously delivering to production. If you have any self-respecting product it'll take you 90% more effort to get from CI (including deployments) to CD where deployments get rolled out automatically using metrics that can reliably indicate that nothing goes wrong during the automated rollout.